anonymizer/anonymizer.h

```diff
@@ -53,6 +53,7 @@
 #include <getopt.h>
 #include <string.h>
 #include <ctype.h>
+#include <regex.h>
```

Comment:
This include does not make sense since regex is used probably only in .c file. (At least according to this diff)